frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -199,6 +198,17 @@ if Device:hasKeys() then
                     G_reader_settings:flipNilOrTrue("enable_back_history")
                 end,
             },
+            {
+                text = _("Invert PageTurn Buttons"),
+                enabled_func = function() return Device:hasKeys() end,
```

Comment:
The whole block that contains this item is already inside `if Device:hasKeys() then`, so this shouldn't be necessary :)

Reply:
Realized that when I checked on my H2O and wondered where the menu had gone ;p.